spark/src/main/scala/geotrellis/spark/matching/RDDHistogramMatching.scala

```diff
@@ -94,7 +94,7 @@ object RDDHistogramMatching {
     sourceHistograms: Seq[Histogram[T1]],
     targetHistograms: Seq[Histogram[T2]]
   ): RDD[(K, MultibandTile)] =
-    rdd.map({ case (key, tile: MultibandTile) =>
+    rdd.map({ case (key, tile) =>
```

Comment:
You don't need extra brackets here: `rdd.map { case (key, tile) =>  (key, HistogramMatching(tile, sourceHistograms, targetHistograms)) }`. This comment can be applied to all changes below btw.

Reply:
I'm going for minimum code change outside of configuration here, would rather not get sucked into format shuffling. Having said that once we get the guild and publishing sorted out we really should look into enforcing a code formatting standard for the library.